sky/skylet/constants.py

```diff
@@ -148,31 +153,29 @@
     'echo "Creating conda env with Python 3.10" && '
     f'conda create -y -n {SKY_REMOTE_PYTHON_ENV_NAME} python=3.10 && '
     f'conda activate {SKY_REMOTE_PYTHON_ENV_NAME};'
+    # Install uv for venv management and pip installation.
+    'which uv >/dev/null 2>&1 || '
+    'curl -LsSf https://astral.sh/uv/install.sh '
+    f'| UV_INSTALL_DIR={SKY_UV_INSTALL_DIR} sh;'
     # Create a separate conda environment for SkyPilot dependencies.
     f'[ -d {SKY_REMOTE_PYTHON_ENV} ] || '
     # Do NOT use --system-site-packages here, because if users upgrade any
     # packages in the base env, they interfere with skypilot dependencies.
     # Reference: https://github.com/skypilot-org/skypilot/issues/4097
-    f'{SKY_PYTHON_CMD} -m venv {SKY_REMOTE_PYTHON_ENV};'
+    f'{SKY_UV_CMD} venv {SKY_REMOTE_PYTHON_ENV};'
```

Comment:
apparently we should be using `venv --seed` for compatibility. The default `uv venv` behavior is apparently like `venv --without-pip`.